src/content/docs/logs/logs-context/net-configure-logs-context-all.mdx

```diff
@@ -567,11 +567,11 @@ You can use our [Serilog](https://serilog.net/) extension to link to your log da
 
     3. Once you have configured the Serilog extension and updated your logging file, there are [several options](https://docs.newrelic.com/docs/logs/forward-logs/enable-log-management-new-relic/#log-forwarding) for forwarding your logs. Here is an example configuration using the infrastructure agent for logs in context:
 
-    ```yml
-    logs:
-      - name: application-log
-        file: C:\logs\log4netExample.log.json # Path to a single log file
-    ```
+      ```yml
+      logs:
+        - name: application-log
+          file: C:\logs\log4netExample.log.json # Path to a single log file
+      ```
     
 #[Documentation for using the infrastructure agent forwarder](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/#file)
```

Comment:
same comment as before re: starting this line with #

Reply:
I corrected those too Barb